### ISSUE TITLE
empty structure not reported,

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#897]
```abap
IF sy-subrc <> 0.
ELSE.
ENDIF.
```
